Doc/howto/unicode.rst

```diff
@@ -152,40 +119,14 @@ problems.
 3. It's not compatible with existing C functions such as ``strlen()``, so a new
    family of wide string functions would need to be used.
 
-4. Many Internet standards are defined in terms of textual data, and can't
-   handle content with embedded zero bytes.
```

Comment:
Actually the larger issue is that many Internet standards depends on ASCII-compatible encoding. You can't compose an email message or a web page using the UTF-32 encoding. XML document can be encoded with UTF-32, but be prepare, that its support is limited (in particular standard Python XML modules don't support it).

Reply:
Right, but it doesn't seem worth it to explain this.  In general I'm removing asides and passing references in this set of edits because every passing references turns out to be subtly incorrect.